hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieHFileInputFormat.java

```diff
@@ -137,15 +55,6 @@ protected HoodieDefaultTimeline filterInstantsTimeline(HoodieDefaultTimeline tim
   @Override
   protected boolean isSplitable(FileSystem fs, Path filename) {
     // This file isn't splittable.
-    return false;
-  }
-
-  public void setConf(Configuration conf) {
-    this.conf = conf;
-  }
-
-  @Override
-  public Configuration getConf() {
-    return conf;
+    return includeLogFilesForSnapShotView();
```

Comment:
Is the splitable property related to whether the log files are included?  If not, the literal should be used for better clarity.

Reply:
Good catch! I think that's actually an IDEA volunteered to do overly smart method extraction. Will revert